src/quacc/recipes/vasp/core.py

```diff
@@ -320,3 +325,72 @@ def non_scf_job(
         additional_fields={"name": "VASP Non-SCF"},
         copy_files={prev_dir: ["CHGCAR*", "WAVECAR*"]},
     )
+
+
+@job
+def freq_job(
+    atoms: Atoms,
+    preset: str | None = "BulkSet",
+    energy: float = 0.0,
+    temperature: float = 298.15,
+    pressure: float = 1.0,
+    vib_kwargs: VibKwargs | None = None,
+    copy_files: SourceDirectory | dict[SourceDirectory, Filenames] | None = None,
+    **calc_kwargs,
+) -> VibThermoSchema:
+    """
+    Run a frequency job and calculate thermochemistry.
+
+    Parameters
+    ----------
+    atoms
+        Atoms object
+    preset
+        Preset to use from `quacc.calculators.vasp.presets`.
+    energy
+        Potential energy in eV. If 0, then the output is just the correction.
+    temperature
+        Temperature in Kelvins.
+    pressure
+        Pressure in bar.
+    vib_kwargs
+        Dictionary of kwargs for the [ase.vibrations.Vibrations][] class.
+    copy_files
+        Files to copy (and decompress) from source to the runtime directory.
+    **calc_kwargs
+        Custom kwargs for the Vasp calculator. Set a value to
+        `None` to remove a pre-existing key entirely. For a list of available
+        keys, refer to [quacc.calculators.vasp.vasp.Vasp][].
+
+    Returns
+    -------
+    VibThermoSchema
+        Dictionary of results, specified in [quacc.schemas.ase.summarize_vib_and_thermo][].
+        See the type-hint for the data structure.
+    """
+    calc_defaults = {
+        "ediff": 1e-8,
+        "ibrion": -1,
```

Comment:
We can remove the ibrion = -1 setting since nsw = 0 is all we need to enforce a static calculation. Internally, quacc will [automatically](https://github.com/Quantum-Accelerators/quacc/blob/90065b4e557dd34cada408bfccaaed9725f8f6ab/src/quacc/calculators/vasp/params.py#L267-L271) drop ibrion and related flags if NSW = 0 anyway.